erts/etc/win32/erl.rc

```diff
@@ -31,10 +31,10 @@ VS_VERSION_INFO VERSIONINFO
     {
         BLOCK "040904e4"
         {
-            VALUE "CompanyName",        "Ericsson\0"
+            VALUE "CompanyName",        "Ericsson AB\0"
             VALUE "FileDescription",    "Erlang\0"
             VALUE "FileVersion",        ERTS_VERSION_STR
-            VALUE "LegalCopyright",     "Ericsson. All rights reserved.\0"
+            VALUE "LegalCopyright",     "Copyright Ericsson AB 2010-2023. All Rights Reserved.\0"
```

Comment:
The 2023 year here is annoying can we script this?
Because we will forget to update this next year.

Reply:
For what it's worth the copyright header in all the other source files is out of date. Some end with 2016, 2021, 2022, etc. I think automating this is outside the scope of this PR.